t/app/controller/waste_bexley_garden.t

```diff
@@ -1642,6 +1642,19 @@ FixMyStreet::override_config {
         is $report->get_extra_metadata('direct_debit_reference'), 'APIRTM-DEFGHIJ1KL', 'Correct payer reference';
         is $report->state, 'confirmed', 'Report is confirmed';
     };
+
+    subtest 'Test paye.net narrative customization' => sub {
+        my $cobrand = FixMyStreet::Cobrand::Bexley->new;
+
+        my $report = FixMyStreet::DB->resultset("Problem")->first;
+        $report->update_extra_field({ name => 'uprn', value => '12345678' });
+
+        my $report_id = $report->id;
+        is $cobrand->waste_get_paye_narrative($report),
+            "Garden Waste Service Payment - Reference: $report_id Contract: 12345678",
+            'Narrative includes reference and contract details';
+    };
+
```

Comment:
Not sure if this unit test is enough, or if I should write a full integration test for the redirect URL code?

Reply:
Interestingly, the test file does already have code to catch call_params and sent_params for any Paye integration calls. I guess having a test that is only a staff member creating a subscription would be a good test to have anyway, not just for this bit? Brent has a staff-subscribe-to-sacks test that e.g. has `is $mech->res->previous->header('Location'), 'http://paye.example.org/...` and checks `$sent_params->items{0}...` so I don't think you'd need to do much more than the steps-to-subscribe for a staff member, as it looks like the stuff to catch the low-level stuff is already present, just not used.